core.cc

```diff
@@ -30,7 +30,7 @@ class Worker: public Nan::AsyncWorker {
         void HandleOKCallback() {
             Nan::HandleScope scope;
 
-            if(!callback->IsEmpty())
+            if(callback->GetFunction()->IsFunction())
                 callback->Call(0, NULL);
         };
```

Comment:
the default implementation doesn't check to see if callback is NULL, so it segfaults if there is no callback